demo/src/index.js

```diff
@@ -0,0 +1,234 @@
+import './index.css'
```

Comment:
Are these demos based on something? I think we can assume people will try them out and change them to suit there needs, so perhaps some comments on what is what?

Reply:
What do you mean by “based on something”?

Reply:
based on another demo.

This is 230 lines of code that does a lot of wiring, which folks will copy/paste and try to modify, but there’s no explanation what anything does

Reply:
It’s based on https://monaco-tailwindcss.js.org / https://monaco-unified.js.org / https://monaco-yaml.js.org. I don’t really feel the need to cross-reference those in comments, but I did add some comments to explain which does what.